### PR TITLE
Fix .count(), update tests

### PR DESCRIPTION
**Pull request by timleslie**
Updates some relationship tests to use the new APIs. Fixes .count() method when called with no args.

### CODE REVIEW

File: packages-next/keystone/src/lib/context/itemAPI.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ export function itemAPIForList(
       }
     },
     async count(args = {}) {
-      const { first, skip, where } = args;
+      const { first, skip = 0, where={} } = args;
       const { listQueryMetaName, whereInputName } = context.gqlNames(listKey);
       const query = `query ($first: Int, $skip: Int, $where: ${whereInputName}) { ${listQueryMetaName}(first: $first, skip: $skip, where: $where) { count }  }`;
```
Comment (mitchellhamilton) [inline packages-next/keystone/src/lib/context/itemAPI.ts:106]:
```suggestion
      const query = `query ($first: Int, $skip: Int! = 0, $where: ${whereInputName}! = {}) { ${listQueryMetaName}(first: $first, skip: $skip, where: $where) { count }  }`;
```

File: packages-next/keystone/src/lib/context/itemAPI.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export function itemAPIForList(
       }
     },
     async count(args = {}) {
-      const { first, skip, where } = args;
+      const { first, skip = 0, where={} } = args;
```
Comment (mitchellhamilton) [inline packages-next/keystone/src/lib/context/itemAPI.ts:104]:
```suggestion
      const { first, skip, where } = args;
```

**Review by mitchellhamilton — CHANGES_REQUESTED** [review]
I don't think the count your count fix will actually fix it(though if you actually tried it, and it worked, that's interesting)

**Review by mitchellhamilton — APPROVED** [review]
I think I was totally wrong.

**Comment by timleslie** [comment]
> I don't think the count your count fix will actually fix it(though if you actually tried it, and it worked, that's interesting)

Yeah, it turned the tests from red to green, pretty sure it's working now (or at least more working than it was :-) )

**Pull request merged**